### ISSUE TITLE
aseismic coefficient vs. seismic coefficient

### ISSUE THREAD

**Issue by cossatot** [GEMScienceTools/oq-mbtk#74]
@julgp @klunk386 @mmpagani @rcgee @kejohnso 

When calculating the MFD from the slip rate, we have a parameter called `aseismic_coefficient` which describes the fraction of the total slip rate that is released seismically.

In the usage of the previous versions of the fault modeling utilities, the total seismic slip rate is

seismic_rate = slip_rate * aseismic_coefficient

However, the word *aseismic* means 'not seismic'.  (Maybe this is only an English usage, and not the same in Italian or Spanish?).  Therefore, I think that the `aseismic_coefficient` should actually describe the fraction of the slip rate that is released *a*seismically, not seismically.  The corresponding seismic slip rate would be

seismic_rate = slip_rate * (1 - aseismic_coefficent)

The default `aseismic_coefficient` in the previous code was 0.9.  This should translate to a new `aseismic_coefficient` of 0.1.

This means that if 10% of a fault's moment accumulation is aseismic (released through creep, afterslip, etc.) then the `aseismic_coefficient` is 0.1 rather than 0.9.

Are we OK with making this change? I believe that doing so is more correct and will be less confusing for users outside of our group, but it means that we have to keep aware of the change when we are modeling.

I already made this change in the new code, but I can change it back.

**Comment by kejohnso** [comment]
Richard, I agree with you and am okay with the change.

**Comment by klunk386** [comment]
I think that the issue is due to the fact that we have simply used the wrong name for that variable. To be consistent with literature (if I'm not mistaking) we should probably use the term "coupling_coefficient" which does exactly what it is supposed to:
"_A coupling coefficient of zero indicates that the fault is creeping at its long-term rate, whereas a value of 1 indicates that it is fully locked and is thus accumulating strain energy that will be released in future earthquakes or episodic creep_"
Changing this representation might be a bit misleading for the standard user, so I suggest to go back to previous version, but definitely change the name.

**Comment by cossatot** [comment]
Valerio, I don't think it's necessarily the same as the coupling coefficient.  As you have stated, the coupling coefficient captures the percentage of the total fault slip that becomes stored as elastic strain energy. This is really important in subduction zones but is only important for a very small percentage of continental faults (certainly far less than 1%). However, some component of the *accumulated* strain is released aseismically as afterslip and episodic creep.

In other words, the aseismic component of total fault slip is the uncoupled component plus afterslip, plus any episodic creep.

**Comment by klunk386** [comment]
OK, fine with it, but then I suggest to extensively explain/discuss the use of this parameter in the README (e.g. in the table of defaults settings) and in the docstring, to avoid any misuse from the user.
